deploy/template.yaml

```diff
@@ -488,6 +488,7 @@ Resources:
           POWERTOOLS_SERVICE_NAME: !Sub initialise-ipv-session-${Environment}
           IPV_SESSIONS_TABLE_NAME: !Select [ 1, !Split [ '/', !GetAtt SessionsTable.Arn ] ]
           SQS_AUDIT_EVENT_QUEUE_URL: !ImportValue AuditEventQueueUrl
+          IPV_CLIENT_OAUTH_SESSIONS_TABLE_NAME: !Ref ClientOAuthSessionsTable
```

Comment:
Not a big deal but could drop the `IPV_` on this

Reply:
+1

Reply:
Removed `IPV_` prefix